01_auth_api/src/db/index.ts

```diff
@@ -0,0 +1,25 @@
+import * as dotenv from 'dotenv';
```

Comment:
I think we can shake import, and take only config with Object destructuring:
import { config } from 'dotenv';

Or see this in [docs](import 'dotenv/config'), looks simpler  than your solution

Reply:
I did this because I thought it would be more readable. When we read the code, we see that this config method is called from dotenv.

Your link to the doc is broken, but I decided to google it and saw this
```typescript
import 'dotenv/config';
``` 
Did you mean this?

Reply:
Yes, this looks clean, also this is written in docs.

 You can leave comments like:  `//SOURCE: link to docs`,  if you think it can be hard to understand